app.js

```diff
@@ -162,3 +162,9 @@ TeamValidator.PM.spawn();
  *********************************************************/
 
 require('./repl').start('app', cmd => eval(cmd));
+
+/*********************************************************
+ * Load the GitHub plugin
+ *********************************************************/
+
+Object.assign(Chat.commands, require('./chat-plugins/github').commands);
```

Comment:
I thought the entire point of having it in Chat was so it was hotpatchable? Well, at least, one of the reasons...